pkg/apis/hub/v1alpha1/api.go

```diff
@@ -29,17 +28,12 @@ import (
 // +kubebuilder:printcolumn:name="PathPrefix",type=string,JSONPath=`.spec.pathPrefix`
 // +kubebuilder:printcolumn:name="ServiceName",type=string,JSONPath=`.spec.service.name`
 // +kubebuilder:printcolumn:name="ServicePort",type=string,JSONPath=`.spec.service.port.number`
-// +kubebuilder:printcolumn:name="CurrentVersion",type=string,JSONPath=`.spec.currentVersion`
+// +kubebuilder:printcolumn:name="Versions",type=string,JSONPath=`.spec.currentVersion`
```

Comment:
The `JSONPath` is incorrect, we want to show the list of versions, but we can't use `.spec.versions`. We will have to store this rendered in the status.

Reply:
I suggest simply removing it as we never intended to display every version here.